site/installing.md

```diff
@@ -220,15 +218,15 @@ Weave Cloud hosts the Scope UI for you, provides secure access control for your
 
 Sign up for a [Weave Cloud account](https://cloud.weave.works/) and obtain a token. Replace `<token>` with your token by running this on the master node or on whatever machine that has `kubectl` configured to authenticate to your Kubernetes cluster:
 
-    $ kubectl apply -f 'https://cloud.weave.works/launch/k8s/weavescope.yaml?service-token=<token>'
+    kubectl apply -n kube-system -f 'https://cloud.weave.works/k8s/scope.yaml?service-token=<token>'
```

Comment:
I'd prefer this to be `--namespace`, because user probably won't know what `-n` really stands for... WDYT?